docs/release-notes/cloud/cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md

```diff
@@ -0,0 +1,24 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023
+
+On the week of November 13th 2023 we removed the option to create, configure, and delete Jira, Slack, and Webhooks integrations for your repositories. These integrations were available on the repository **Settings**, tab **Integration**.
+
+<!--TODO CY-7215 Add rationale for this removal-->
+
+## Removal of existing integrations
+
+On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->
+
+<!--TODO CY-7215 Impacts?-->
+
+## If you are using Jira, Slack, or Webhooks repository integration
+
+Your existing integrations will keep working until MM DD YYYY, but you are no longer able to configure, or delete them on the Codacy app.
+
+<!--TODO CY-7215 Instructions for customers?-->
```

Comment:
@RaquelHipolito @mrfyda For customers using these integrations, are there any particular steps they can or should do?

Reply:
no actions required

if you want to add a note, we'll also delete any authentication data that is left from the integrations once they are removed

if they have a critical workflow relying on these integrations, please let us know what so that we can advise on the best alternative